Src/ILGPU/Runtime/ExchangeBuffer.cs

```diff
@@ -14,6 +14,7 @@
 using ILGPU.Util;
 using System;
 using System.Diagnostics;
+using System.Net;
```

Comment:
I am wondering about this import. Is this required?

Reply:
No, I guess I never removed it.